spec/models/storage_location_spec.rb

```diff
@@ -145,6 +145,14 @@
       end
     end
 
+    describe "#total_inventory_fair_market_value" do
+      it 'returns the total value of the items in it' do
```

Comment:
Can you add a test for the behavior where one of the value_in_cents is set to nil?

Reply:
actually, the default value of `value_in_cents` is set to 0.